src/commands/install-deps.yml

```diff
@@ -45,22 +38,35 @@ steps:
           APP_BUNDLER_VERSION=<<parameters.bundler-version>>
         fi
 
-        if ! echo $(bundle version)  | grep -q $APP_BUNDLER_VERSION; then
+        if ! echo $(bundle version) | grep -q $APP_BUNDLER_VERSION; then
           echo "Installing bundler $APP_BUNDLER_VERSION"
           gem install bundler:$APP_BUNDLER_VERSION
         else
           echo "bundler $APP_BUNDLER_VERSION is already installed."
         fi
 
+        test -f Gemfile.lock || bundle lock
```

Comment:
I don't think we want to do `bundle lock`. Poking around with it, `bundle lock` will generate a new lockfile with the latest versions of everything, but `bundle check` checks for gems already on the system that satisfy the `Gemfile`. If no `Gemfile.lock` exists, `bundle check` will generate one based on the gems already on the system.